### ISSUE TITLE
ck2cti is not working in Cantera 2.3.0

### ISSUE THREAD

**Issue by dxtptse** [Cantera/cantera#466]
Hello everyone,

I have Cantera 2.3.0 installed im my University's computer. It was installed from the repository, all packages were downloaded and installed (cantera-dev, cantera-common, cantera-python, cantera-python3, scons, numpy, python, etc). The package works well when I test Python programs, however, I cannot use ck2cti to convert files from CHEMKIN to Cantera format. It has to be run from command line, so, when I run "ck2cti --input=<mech-file> --thermo=<thermo-file> --transport=<tpt-file>", I get "ck2cti: command not found".
What is strange in this case is that in my personal computer, running Ubuntu 16.04 LTS, with Cantera 2.1.2, it works flawlessly. I also was able to reproduce the same situation in a virtual machine running the same Xubuntu version  (by the way, I tried to install Cantera from the source code first, but scons give me an /fmt/ error, which I did not try to solve after all).
I don't know if I need to run a source $CANTERA_BASH or something, or if it is a real bug (which is what I believe). I am sending attached an example of the kind of files I need to convert (the famous GRI 3.0 scheme).

Thank you!
[grimech30.txt](https://github.com/Cantera/cantera/files/1173514/grimech30.txt)
[thermo30.txt](https://github.com/Cantera/cantera/files/1173515/thermo30.txt)
[transport.txt](https://github.com/Cantera/cantera/files/1173513/transport.txt)

### Cantera version
2.3.0
### Operating System
Xubuntu 16.04.1 LTS
### Python/MATLAB version
2.7/3/3.5, no MATLAB
### Expected Behavior
It should be able to convert files from CHEMKIN to Cantera format
### Actual Behavior
ck2cti is not recognised as a command.
### Steps to reproduce
ck2cti --input=grimech30.txt --thermo=thermo30.txt --transport=transport.txt

**Comment by bryanwweber** [comment]
Thanks for the report! How, precisely, did you install Cantera? Can you post the value of your `PATH` environment variable?

By the way, you can find instructions for compiling Cantera here: http://cantera.org/docs/sphinx/html/compiling/index.html If you downloaded the source code from Sourceforge, there are a few packages (including fmt) that you may have to install on the system, so I recommend downloading the source code via git clone instead.

**Comment by dxtptse** [comment]
About the $PATH variable: In my own computer, now after cantera 2.3.0 update (presenting the same issues): "/home/user/bin:/home/user/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin", plus others related to OpenFOAM and Java. There is no mention to any cantera /bin folder. Maybe that's the origin of the problem. I have tried "python -m cantera.ck2cti" and it worked, but I would like to know how to fix it, and maybe avoid further issues.

About the installation: Cantera was installed by another person, who installs engineering software in our department. I don't have "sudo" credentials in my computer there. But he told me that he had installed all the packages through "aptitude", which was what I did too in a virtual machine, at home, facing the same issue. When I tried through the source code at that time, I just downloaded the file from Sourceforge and ran "scons build".

I am a bit of a "newbie" in Linux, I think that was the problem with the installation through the source code. I also found it a bit hard to understand the step by step described in the Cantera page. I tried again, uninstalling the repository installation and replacing it with the source code in my VM (scons build + sudo scons install in cantera folder), and everything is working fine, including ck2cti. Curiously, I get the same response from $PATH, whithout any cantera reference. But the issue with the "aptitude" installation persists, which is what causes me trouble in the university.

Thank you for helping!

**Comment by bryanwweber** [comment]
Your PATH variable looks fine, the script should be installed in `/usr/local/bin` or `/usr/bin` I think, so there shouldn't be any reference to Cantera anywhere. Aptitude is certainly a supported method of installation, so we'll try to get this sorted out. I'm going to ping @speth for help here, and I'll try to reproduce it myself when I get the chance.

I'm glad that the source build worked out in the end, and its an important data point that everything works when you install from source. Are there any instructions that we could clarify in the step-by-step? This is the perpetual problem - the documentation is written by someone who already figured it out...

**Comment by speth** [comment]
This is a regression introduced when we switched to using `setuptools` to install the Python module (b979cea3d2889e274775adf8b234f90afec3b37c) and the installation of the full `ck2cti` and `ctml_writer` scripts to `/usr/bin/` was removed. I have uploaded new versions of the 2.3.0 Ubuntu packages that fix this problem for Xenial and Zesty (Launchpad no longer accepts build requests for Yakkety packages). If you update your installed packages, this should work as expected.
